pyproject.toml

```diff
@@ -10,6 +10,8 @@ Flask = "^2.0.3"
 python-dotenv = "^0.14.0"
 requests = "^2.28.1"
 gunicorn = "^20.1.0"
+pymongo = "^4.3.3"
+mongomock = "^4.1.2"

```

Comment:
Minor, but it would be nice to put this under our dev dependencies below just to keep the number of production libraries as low as possible, reducing risk and keeping our image size down (admittedly probably by a tiny amount!)